recipes/libmysqlclient/all/conanfile.py

```diff
@@ -151,11 +151,13 @@ def _configure_cmake(self):
     def build(self):
         self._patch_files()
         cmake = self._configure_cmake()
-        cmake.build()
+        with tools.environment_append(RunEnvironment(self).vars):
```

Comment:
See [the docs](https://docs.conan.io/en/latest/reference/tools.html#tools-run-environment)
```suggestion
        with tools.run_environment(self):
```